client/dashboard/app/router.tsx

```diff
@@ -209,14 +209,40 @@ const siteMonitoringRoute = createRoute( {
 const siteLogsRoute = createRoute( {
 	getParentRoute: () => siteRoute,
 	path: 'logs',
+} );
+
+const siteLogsIndexRoute = createRoute( {
+	getParentRoute: () => siteLogsRoute,
+	path: '/',
+	beforeLoad: ( { params } ) => {
+		throw redirect( { to: `/sites/${ params.siteSlug }/logs/php` } );
+	},
+} );
+
+const siteLogsPhpRoute = createRoute( {
+	getParentRoute: () => siteLogsRoute,
+	path: 'php',
 } ).lazy( () =>
 	import( '../sites/logs' ).then( ( d ) =>
-		createLazyRoute( 'site-logs' )( {
-			component: d.default,
+		createLazyRoute( 'site-logs-php' )( {
+			component: ( props ) => <d.default { ...props } key="php" />,
```

Comment:
Instead of key, I guess it would be better to pass something like the `logType` here. Also, I think we can remove `{ ...props }` here as it's not used by the component 🙂

Reply:
Good point. I'll refactor in the next PR.